polaris/ocean/resolution.py

```diff
@@ -0,0 +1,20 @@
+def resolution_to_subdir(resolution):
+    """
+    Convert a resolution to a subdirectory name
+
+    Parameters
+    ----------
+    resolution : float
+        The resolution in km
+
+    Returns
+    -------
+    res_str : str
+        The resolution as a string for use as a subdirectory
+
+    """
+    if resolution >= 1.:
+        res_str = f'{resolution:g}km'
```

Comment:
```suggestion
        res_str = f'{resolution:g}km'
    elif resolution < 0.001:
        res_str = f'{resolution * 1.e5:g}cm'
```
This is needed for the `dam_break` case.

Reply:
Sure, sounds good.